modules/gpt3module.py

```diff
@@ -180,74 +194,64 @@ async def gpt3_chat(self, message):
             im = default_italics_mark
 
         if self.openai.is_channel_allowed(message):
-            self.log.info(self.class_name, msg="sending chat prompt to openai", engine=engine)
+            self.log.info(
+                self.class_name, msg="sending chat prompt to openai", engine=engine
+            )
             response = self.openai.get_response(engine, prompt, logit_bias)
             self.log.info(self.class_name, response=response)
             if response != "":
-                return Response(confidence=10, text=f"{im}{response}{im}", why="OpenAI GPT-3 made me say it!")
-
-            # If OpenAI Failed, redo work and try GooseAI.
-            self.log.critical(self.class_name, msg="OpenAI Failed! Trying GooseAI!")
-            engine = self.get_engine(message, force_goose=True)
-            forbidden_tokens = self.get_forbidden_tokens(message.channel, engine)
-            self.log.info(self.class_name, forbidden_tokens=forbidden_tokens, engine=engine)
-            logit_bias = {
-                self.tokenize(engine, "*"): -100,
-                self.tokenize(engine, "**"): -100,
-                self.tokenize(engine, "***"): -100,
-                self.tokenize(engine, " *"): -100,
-                self.tokenize(engine, " **"): -100,
-                self.tokenize(engine, " ***"): -100,
-            }
-            for forbidden_token in forbidden_tokens:
-                logit_bias[forbidden_token] = -100
-
-
-        self.log.info(self.class_name, msg="sending chat prompt to goose.ai", engine=engine)
-        response = self.gooseai.get_response(engine, prompt, logit_bias)
-        self.log.info(self.class_name, response=response)
-        if response != "":
-            return Response(confidence=10, text=f"{im}{response}{im}", why="GooseAI GPT-3 made me say it!")
+                return Response(
+                    confidence=10,
```

Comment:
ah, should this be confidence 10? is the confidence to decide whether or not to call GPT somewhere else and this is just for when we get a response? or can it conflict with other modules confidence 10? I would prefer non-GPT responses (e.g. about accepting Coda review or stuff) and lower this one to 9 if there is a risk of conflicts 🤔

Reply:
well, **theoretically** this shouldn't interfere, but there's no harm in doing it just in case